fedn/fedn/clients/reducer/restservice.py

```diff
@@ -803,8 +803,7 @@ def config_download():
 controller:
     discover_host: {discover_host}
     discover_port: {discover_port}
-    {chk_string} 
-    {token_string}""".format(network_id=network_id,
+    {chk_string}""".format(network_id=network_id,
```

Comment:
Thanks for the fix. Could you explain what this does in the description?

Reply:
It makes it work.. :-) I was a bit sloppy in the previous PR and it was not caught by the CI, but there was a but here that popped up in the manual deployment.